packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/Databases.tsx

```diff
@@ -37,38 +35,80 @@ export default function Container() {
 }
 
 function DatabaseList(props: State) {
+  const {
+    dbs = [],
+    connect,
+    fetchAttempt,
+    agentFilter,
+    pageCount,
+    customSort,
+    prevPage,
+    nextPage,
+    updateQuery,
+    onAgentLabelClick,
+    disabledRows,
+    updateSearch,
+    emptyTableText,
+  } = props;
+
   return (
     <>
-      {props.syncStatus.status === 'failed' && (
-        <Danger>{props.syncStatus.statusText}</Danger>
+      {fetchAttempt.status === 'error' && (
+        <Danger>{fetchAttempt.statusText}</Danger>
       )}
-      <Table
-        data={props.dbs}
-        columns={[
-          {
-            key: 'name',
-            headerText: 'Name',
-            isSortable: true,
-          },
-          {
-            key: 'labelsList',
-            headerText: 'Labels',
-            render: renderLabelCell,
-          },
-          {
-            altKey: 'connect-btn',
-            render: db => (
-              <ConnectButton
-                dbUri={db.uri}
-                protocol={db.protocol as GatewayProtocol}
-                onConnect={dbUser => props.connect(db.uri, dbUser)}
-              />
-            ),
-          },
-        ]}
-        pagination={{ pageSize: 15, pagerPosition: 'bottom' }}
-        emptyText="No Databases Found"
+      <SearchPanel
+        updateQuery={updateQuery}
+        updateSearch={updateSearch}
+        pageCount={pageCount}
+        filter={agentFilter}
+        showSearchBar={true}
+        disableSearch={disabledRows}
       />
+      <DarkenWhileDisabled disabled={disabledRows}>
+        <Table
+          data={dbs}
+          columns={[
+            {
+              key: 'name',
+              headerText: 'Name',
+              isSortable: true,
+            },
+            {
+              key: 'desc',
+              headerText: 'Description',
+              isSortable: true,
+            },
+            {
+              key: 'type',
```

Comment:
Thanks for adding this, but it needs to be formatted before displaying. Please take a look at https://github.com/gravitational/webapps/blob/master/packages/teleport/src/services/databases/makeDatabase.ts

Reply:
Done 👍